devel/howdoi/Portfile

```diff
@@ -2,35 +2,28 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
+PortGroup           github 1.0
 
+github.setup        gleitz howdoi 2.0.19 v
 name                howdoi
-version             1.1.9
 categories          devel python
-platforms           darwin
 license             MIT
 supported_archs     noarch
-
-python.default_version 27
-
 maintainers         nomaintainer
 
-description         instant coding answers via the command line
+description         Instant coding answers via the command line
 long_description    {*}${description}
-homepage            https://pypi.python.org/pypi/howdoi
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
-
-distname            ${python.rootname}-${version}
 
-checksums           rmd160  9c1fdc3dfd03fd717b4978b05928bc0d565f6f10 \
-                    sha256  2b4a0ebc39c9e0489a17ef73e3eec7dd4a2975e07b995406887c639481da2bb5
+checksums   rmd160  6a2668e5e805a6569b5ea81653aecfc1c86290cf \
+            sha256  c8e3ff3461fbc7a09d36c2ac4e743f548215b47848c4ac2ad0716f96be163517 \
+            size    2516102
 
-depends_build-append \
-    port:py${python.version}-setuptools
+python.default_version 39
 
-depends_lib-append \
-    port:py${python.version}-requests \
-    port:py${python.version}-requests-cache \
-    port:py${python.version}-pygments \
-    port:py${python.version}-pyquery
+depends_build-append   port:py${python.version}-setuptools
+depends_lib-append     port:py${python.version}-requests \
+                       port:py${python.version}-requests-cache \
+                       port:py${python.version}-pygments \
+                       port:py${python.version}-pyquery
 
-livecheck.type      pypi
+livecheck.type      none
```

Comment:
this is not correct; you do want to have the livecheck on here

Reply:
> this is not correct; you do want to have the livecheck on here

Ok, no problem. What I should put in place of `pypi`? Or `pypi` gonna still work with Github?

Reply:
this is the default set by the python PG so no need for this line